pcsx2/CDVD/CDVD.cpp

```diff
@@ -1222,6 +1222,7 @@ static void cdvdWrite04(u8 rt) { // NCOMMAND
 				Console.WriteLn( Color_Gray, L"CdRead: Reading Sector %07d (%03d Blocks of Size %d) at Speed=%dx",
 					cdvd.SeekToSector, cdvd.nSectors,cdvd.BlockSize,cdvd.Speed);
 
+			cdvd.PwOff = 1 << Irq_DataReady;
```

Comment:
Possibly the change could go in `cdvdReadInterrupt()`. The Street Fighter block dump still works, and  I think it makes more sense that way (the data actually is ready if I'm reading the code correctly).
```
diff --git a/pcsx2/CDVD/CDVD.cpp b/pcsx2/CDVD/CDVD.cpp
index 9b0da9a..8ecd9fe 100644
--- a/pcsx2/CDVD/CDVD.cpp
+++ b/pcsx2/CDVD/CDVD.cpp
@@ -890,7 +890,7 @@ __fi void cdvdReadInterrupt()
 
 	if (--cdvd.nSectors <= 0)
 	{
-		cdvd.PwOff |= 1<<Irq_CommandComplete;
+		cdvd.PwOff |= (1 << Irq_CommandComplete) + (1 << Irq_DataReady);
 		psxHu32(0x1070)|= 0x4;
 
 		HW_DMA3_CHCR &= ~0x01000000;
```